frontend/src/i18n/locales/keys.test.ts

```diff
@@ -0,0 +1,85 @@
+import { error } from 'console';
```

Comment:
I feel there should be a description of this test that's a bit more 
- near the top
- descriptive when it fails. Currently the test is called "Translation keys", but it should give people a clue what is happening and what they need to do right away when it fails in the error message given in the failure. `npm run test` should describe the problem and what to do to fix it.